browser_patches/firefox/patches/bootstrap.diff

```diff
@@ -638,7 +638,7 @@ index 48af2c30f3604a94e6bc1da7848aed3c8ac11c1e..f9c1df0b36b3c9342b81b4c2c7bc7aec
      IgnoreRFP aIgnoreRFP) const {
 +  auto* docShell = static_cast<nsDocShell*>(GetDocShell());
 +  nsIDocShell::ColorSchemeOverride colorScheme;
-+  if (docShell->GetColorSchemeOverride(&colorScheme) == NS_OK &&
++  if (docShell && docShell->GetColorSchemeOverride(&colorScheme) == NS_OK &&
```

Comment:
NB: This change avoids crashing Firefox when there is a
`prefers-color-scheme` query in an SVG favicon, and assumes
it's OK if Playwright's `colorScheme` emulation does not
apply to the favicon context. I think the favicon is outside the scope of Playwright and most testing; however, if not, is there a way to invoke `GetColorSchemeOverride` on some other object that would give us what we want in this case?